portfolio.html

```diff
@@ -82,5 +82,237 @@ <h2>I'm Amrendra Glad to see you!</h2>
+          <div class="aboutconnect">LET'S CONNECT</div>
+          <ul class="aboutsocial-icons">
+            <li>
+              <a
+                href="https://twitter.com/amrendrak_"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-twitter"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/amrendraakumar/"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-linkedin-in"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://github.com/amrendrakind"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><i class="fab fa-github"></i
+              ></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <button type="button" class="btnseeproject">Get my resume</button>
+
+      <!-- //////////// -->
+
+      <div class="acordianwrapper">
+        <button class="accordion">Languages</button>
+        <div class="panel">
+          <div class="languagename">
+            <img src="/Ellipse_JS.png" alt="JSIcon" />
+            JavaScript
+          </div>
+          <div class="languagename">
+            <img src="./Ellipse_HTML.png" alt="HTMLIcon" />
+            HTML
+          </div>
+          <div class="languagename">
+            <img src="./Ellipse_CSS.png" alt="CSSIcon" />
+            CSS
+          </div>
+        </div>
+
+        <button class="accordion">Frameworks</button>
+        <div class="panel">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
+        </div>
+
+        <button class="accordion">Skills</button>
+        <div class="panel">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
+        </div>
+
+        <script>
+          var acc = document.getElementsByClassName("accordion");
+          var i;
+          for (i = 0; i < acc.length; i++) {
+            acc[i].addEventListener("click", function () {
+              this.classList.toggle("active");
+              var panel = this.nextElementSibling;
+              if (panel.style.maxHeight) {
+                panel.style.maxHeight = null;
+              } else {
+                panel.style.maxHeight = panel.scrollHeight + "px";
+              }
+            });
+          }
+        </script>
```

Comment:
- [ ] [OPTIONAL] Nice implementation of JS 👍 👍  You can implement it in the future, on JS need milestones.